src/main/java/ua/cv/tim/dao/hibernate/UserDaoImpl.java

```diff
@@ -43,9 +56,7 @@ public User getWithRolesById(String id) {
     }
 
     public User getByMail(String mail, String uuid) {
-        System.out.println("Dao get by email");
-        System.out.println("uuid = : " + uuid);
-
+        logger.info("UserDao.getByMail  started mail is {} uuid is {} ", mail,uuid);
         Session session = getCurrentSession();
```

Comment:
I think better do not create Session instance and use getCurrentSession() method instead when create Query instance